synthesizer/sed.py

```diff
@@ -261,7 +261,7 @@ def get_broadband_fluxes(self, fc, verbose=True):  # broad band flux/nJy
                            'the same as the SED wavelength grid.'))
 
             # Calculate and store the broadband flux in this filter 
-            bb_flux = f.apply_filter(self._lnu, nu=self.nuz) * nJy
+            bb_flux = f.apply_filter(self._fnu, nu=self.nuz) * nJy
```

Comment:
this feels like an unrelated change, were broadband luminosities not correctly calculated previously?